File: codes/283-move-zeroes/move-zeroes.py
class Solution(object):
    def moveZeroes(self, nums):
        """
        :type nums: List[int]
        :rtype: None Do not return anything, modify nums in-place instead.
        """
        zero=[]
        list=[]
        for num in nums:
            if num == 0:
                zero.append(num)
            else:
                list.append(num)
                
        combined = list+zero
        nums[:]=combined
        return nums

File: codes/283-move-zeroes/move-zeroes.cpp
#include<vector>
using namespace std;
class Solution {
public:
    void moveZeroes(vector<int>& nums) {
    vector <int> list={};

    for (int num:nums){
        if(num!=0){
            list.push_back(num);
        }
    }

    for (int i=0;i<list.size();i++){
        nums[i]=list[i];
    }
    for(int i= list.size();i<nums.size();i++){
        nums[i]=0;
    }
    }
};